ReSharper.FSharp/src/FSharp.Psi.Features/src/CodeCleanup/FantomasSettingsConverterProvider.fs

```diff
@@ -0,0 +1,43 @@
+module JetBrains.ReSharper.Plugins.FSharp.Psi.Features.CodeCleanup.FSharpEditorConfig
+
+open System
+open JetBrains.Application
+open JetBrains.Application.Settings
+open JetBrains.Application.Settings.Calculated.Implementation
+open JetBrains.DataFlow
+open JetBrains.Diagnostics
+open JetBrains.Lifetimes
+open JetBrains.ReSharper.Plugins.FSharp.Services.Formatter
+open JetBrains.ReSharper.Psi.EditorConfig
+
+type FantomasSettingsConverter(fantomasSettingsEntry: SettingsIndexedEntry) =
+  let fSharpPrefix = "fsharp_"
```

Comment:
Off-topic: this is a wrong indent size for F# files in this project. @DedSec256 How did you get this indentation?